addons/editor-theme3/addon.json

```diff
@@ -364,6 +359,7 @@
     },
     {
       "name": "Comments",
+      "description": "When using colored text on a black/white background this setting affects the borders of standalone comments.",
```

Comment:
If we add support for showing descriptions conditionally in the future, this should only be shown if the text color is set to colored.